tasks/db_tasks.py

```diff
@@ -85,12 +89,28 @@ def seed_dev_db(c, fundround=None, appcount=None):
                 "COFR2W2": {
```

Comment:
As we've got this mapping, could we use in the `import_from_application` script so we can just pass a fund/round eg COFR2W2 on the command line and it gets the IDs from here? More readable for testing and dev as well as when we run it for real. Also allow specifying of full IDs tho.